pkg/rancher-components/src/components/Form/Radio/RadioGroup.test.ts

```diff
@@ -0,0 +1,37 @@
+import { mount } from '@vue/test-utils';
+import { RadioGroup } from './index';
+
+describe('component: RadioGroup', () => {
+  describe('when disabled', () => {
+    /**
+     * NOTE: Current setup does not match guidelines for the test of scoped slots
```

Comment:
This test looks very similar to the recommendation for v1 of Vue Test Utils, is there something that I'm missing in this note?

https://v1.test-utils.vuejs.org/api/options.html#scopedslots

Reply:
That's a very good point, I did not notice that they changed also this.